server/utils/cache.js

```diff
@@ -0,0 +1,37 @@
+const NodeCache = require('node-cache');
+const CACHE_TTL_ONE_MINUTE = 1 * 60;
```

Comment:
Should the cache time be a configurable ENV value? (if so, I would move `cache.js` to `configure/cache.js`